tests/example_test.py

```diff
@@ -0,0 +1,5 @@
+import pytest
```

Comment:
You can delete this whole file.  It's just a placeholder form the cookiecutter template